gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcTransportChannel.java

```diff
@@ -89,7 +89,7 @@ public boolean awaitTermination(long duration, TimeUnit unit) throws Interrupted
   }
 
   @Override
-  public void close() throws Exception {
+  public void close() {
```

Comment:
To avoid making this a breaking change, what if you declared that close() throws some *RuntimeException*? That way, code that's already catching Exception won't break, but others won't be required to wrap this in a try/catch.

Reply:
I think there is still the possibility of breaking people who have *implemented* the interfaces which will change, for example `BackgroundResource` - if they have implemented `close()` with `throws Exception`, then they will be broken if we remove it (or change it to `RuntimeException`) on the underlying interface.

This feels like a fairly obscure breakage though (does *anyone* outside of our library implement `BackgroundResource`, or extend any of the relevant classes (and override `close()` and include the `throws Exception` clause?)), hence my question about how to treat this.

Reply:
Good point on those implementing the interfaces -- hadn't thought of that.

I'm not sure about what existing users are doing for other APIs, so I'll defer to you and Garrett on that question.